### PR TITLE
Mobile view

### PR DESCRIPTION
**Pull request by uno36**
added readme
configured linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="styles.css">
+<body>
+    <header class="header">
+    <nav class="navbar">
+        <a href="#" class="nabar-brand"> My logo</a>
```
Comment (zieeco) [inline index.html:13]: - Hey, please, kindly move the logo out of the nav. it is fine for the logo to be inside the header. let's improve on this to improve our page layout.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <i class="fa fa-bars" class="menu-icon"></i>
+        
+    </nav>
+    </header>
+
+    <section class="Mobile-view">
+        <div class="mobile-container">
+            <div class="info">
+                <p class="intro">Hey There. I'm alex.</p>
+                <p class="skill">I'm a Software Developer</p>
```
Comment (zieeco) [inline index.html]: - Hey, this is a subheading to the main heading of your page's content and as such should be from an h2-h6 tag. let's improve our semantic usage.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <a href="#" class="nabar-brand"> My logo</a>
+        <i class="fa fa-bars" class="menu-icon"></i>
+        
+    </nav>
+    </header>
+
+    <section class="Mobile-view">
+        <div class="mobile-container">
+            <div class="info">
+                <p class="intro">Hey There. I'm alex.</p>
```
Comment (zieeco) [inline index.html]: - Hey, this is the main heading of your page's content and as such should be an h1 tag. let's improve our semantic usage.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                        <li><a href=""><i class="fa fa-twitter"></i></a></li>
+                        <li><a href=""><img src="images/Vector.png" alt="medium-icon"></a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="first-page-deco-img"></div>
+        </div>
+    
+
+
```
Comment (zieeco) [inline index.html]: - Hey, let's remove this empty spaces to format our HTML file in a professional way.

Comment (zieeco) [reply index.html]: ### README FILE


- [ ] The license link is broken, lets fix this by adding the appropriate missing license to make the readme more professional.
- [ ] let's remove this placeholder text as it's been added to guide you on making updates to the readme and is not expected to be in the readme file after you've made an update to the readme. see the image:
Screenshot from 2023-03-20 19-50-52

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @uno36 ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @uno36 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

No linter errors.✔️
Nice design. ✔️
Descriptive PR. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**